### PR TITLE
Enable cluster state bundle in distributor.

### PR DESCRIPTION
**Pull request by toregge**
@geirst, @vekterli: please review

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

File: storage/src/vespa/storage/distributor/pendingclusterstate.cpp [hunk 1]
```diff
@@ -32,8 +32,8 @@ PendingClusterState::PendingClusterState(
         api::Timestamp creationTimestamp)
     : _cmd(newStateCmd),
       _requestedNodes(newStateCmd->getSystemState().getNodeCount(lib::NodeType::STORAGE)),
-      _prevClusterState(clusterInfo->getClusterState()),
-      _newClusterState(newStateCmd->getSystemState()),
+      _prevClusterStateBundle(clusterInfo->getClusterStateBundle()),
+      _newClusterStateBundle(newStateCmd->getSystemState()),
```
Comment (vekterli) [inline storage/src/vespa/storage/distributor/pendingclusterstate.cpp]: Should this be `newStateCmd->getClusterStateBundle()` instead? Currently constructed from singular `ClusterState`

**Review by vekterli — APPROVED** [review]
:+1: See question. Doesn't really matter before we actually send down more bucket space states with the RPC.

**Review by vekterli — APPROVED** [review]
:+1:

**Pull request merged**